### PR TITLE
EVG-12573 Add distros resolver

### PR DESCRIPTION
**Pull request by khelif96**

### CODE REVIEW

File: graphql/resolvers.go [hunk 1]
```diff
@@ -1114,6 +1114,34 @@ func (r *queryResolver) HostEvents(ctx context.Context, hostID string, hostTag *
 	return &hostevents, nil
 }
 
+func (r *queryResolver) Distros(ctx context.Context, onlySpawnable bool) ([]*restModel.APIDistro, error) {
+	apiDistros := []*restModel.APIDistro{}
+
+	var distros []distro.Distro
+	if onlySpawnable == true {
```
Comment (tgrander) [inline graphql/resolvers.go:1121]: is the ` == true` needed?

File: graphql/schema.graphql [hunk 2]
```diff
@@ -233,6 +234,13 @@ type DistroInfo {
   user: String
 }
 
+type Distro {
+  name: String 
+  userSpawnAllowed: Boolean
+  workDir: String 
+  user: String 
+  isVirtualWorkStation: Boolean!
+
```
Comment (tgrander) [inline graphql/schema.graphql:243]: there's no closing bracket?

Comment (khelif96) [reply graphql/schema.graphql:243]: thanks i must have accidentally back spaced it before i committed.

File: graphql/tests/distros/data.json [hunk 3]
```diff
@@ -0,0 +1,266 @@
+{
```
Comment (john-m-liu) [inline graphql/tests/distros/data.json:1]: Could you try using the data scrambler on this file by typing `make scramble file=graphql/tests/distros/data.json`? It should work now. You can undo it if doing so requires you to change too many other tests/files but it'd be nice to start using it

Comment (khelif96) [reply graphql/tests/distros/data.json:1]: image
I think i broke it

**Comment by khelif96** [comment]
test-rest-model is failing but its also failing on the base branch so gonna merge anyways

**Pull request merged**